<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicalRecords</class>
 <widget class="QDialog" name="MedicalRecords">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_record">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Medical Records</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient Name: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_patientNumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_diagnosis">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Diagnosis:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prescription">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prescription: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_treatment">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Treatment: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_patientName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>49</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_patientNumber">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>89</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_diagnosis">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prescription">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_treament">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addRecord">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ediRecord">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_insurance">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Insurance:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_insurance">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
